staging/src/k8s.io/pod-security-admission/admission/api/v1/doc.go

```diff
@@ -1,5 +1,5 @@
 /*
```

Comment:
don't move the existing file, keep the v1beta1 defaults_test.go file in place and create a new v1 defaults_test.go file